packages/devtools_app/pubspec.yaml

```diff
@@ -26,6 +26,9 @@ dependencies:
   codemirror: ^0.5.10
   collection: ^1.14.11
   devtools_server: ^0.1.13
+#  devtools_shared: 0.0.1
+  devtools_shared:
+    path: ../devtools_shared
```

Comment:
we should use a dependency override instead of a path dependency.

Reply:
Eliminating the path part (debug until package was published).